pf-keys/definitions/geom.yaml

```diff
@@ -1480,6 +1488,151 @@ TopoSlopesY:
         domains:
           DoubleValue:
 
+# -----------------------------------------------------------------------------
+# ChannelWidth
+# -----------------------------------------------------------------------------
+# ChannelWidth missing from manual and grep
```

Comment:
The users manual should be updated with the new keys.

Reply:
We will generate the manual again once this PR gets merged to main. I think that these keys will appear automatically since they are included in the yaml file?

Reply:
The documentation RST files are not automatically generated from the YAML documentation.